site/app/templates/GlobalHeader.twig

```diff
@@ -21,6 +21,14 @@
             })
         }
     </script>
+    <script>
```

Comment:
Need to move all this code into a document load listener, or it won't be able to manipulate the element.